### PR TITLE
Imkafka: Added multithreaded support for kafka consumers

### PR DESCRIPTION
**Pull request by alorbach**
Imkafka now creates a worker thread for each configured kafka input. 
Added two tests with 4 and 8 imkafka input sources.

### CODE REVIEW

File: plugins/imkafka/imkafka.c [hunk 1]
```diff
@@ -692,53 +707,53 @@ ENDfreeCnf
...
 		 * against totally hogging the CPU if the users selects a polling interval
 		 * of 0 seconds. It doesn't hurt any other valid scenario. So do not remove.
-		 * rgerhards, 2008-02-14
 		 */
 		if(glbl.GetGlobalInputTermState() == 0)
 			srSleep(0, 100000);
-	} while(glbl.GetGlobalInputTermState() == 0);
+	}
+
+	// TODO: Wait for Worker threads to be finished!
```
Comment (rgerhards) [inline plugins/imkafka/imkafka.c]: I think this comment no longer applies (see `afterRun`)? If so, please remove. Else we need to do the implementation.

File: tests/imkafka_multi_single.sh [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+
+echo \[sndrcv_kafka.sh\]: stop kafka instances
+. $srcdir/diag.sh stop-kafka
+. $srcdir/diag.sh stop-zookeeper
+
+# Do the final sequence check
+seq_check 1 $TESTMESSAGESFULL -d
+
+echo success
+exit_test
```
Comment (rgerhards) [inline tests/imkafka_multi_single.sh]: The lines needs to be properly terminated (LF)

File: tests/imkafka_multi_many.sh [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+. $srcdir/diag.sh stop-kafka '.dep_wrk3'
+. $srcdir/diag.sh stop-zookeeper '.dep_wrk1'
+. $srcdir/diag.sh stop-zookeeper '.dep_wrk2'
+. $srcdir/diag.sh stop-zookeeper '.dep_wrk3'
+
+# Do the final sequence check
+seq_check 1 $TESTMESSAGESFULL -d
+
+echo success
+exit_test
```
Comment (rgerhards) [inline tests/imkafka_multi_many.sh]: The lines needs to be properly terminated (LF)

File: plugins/imkafka/imkafka.c (hunk 1 above)
Comment (alorbach) [reply plugins/imkafka/imkafka.c]: oh indeed, will remove

File: tests/imkafka_multi_single.sh (hunk 2 above)
Comment (alorbach) [reply tests/imkafka_multi_single.sh]: converted into Unix LF

File: tests/imkafka_multi_many.sh (hunk 3 above)
Comment (alorbach) [reply tests/imkafka_multi_many.sh]: converted into Unix LF

**Pull request merged**